examples/flux/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM fluxrm/flux-sched:focal
+# docker build -t flux-snake .
+# docker run -it flux-snake
+USER root
+ENV PATH=/opt/conda:/bin:$PATH
+RUN pip install git+https://github.com/vsoch/snakemake@add/flux-executor
```

Comment:
Should this be pointing at the real repo and not your clone? Or were you thinking you would make a PR to change this line once this PR is merged?

Reply:
I had this here just so a tester of the PR could quickly use and test - I can either change this just before merge, or right after (the first is probably better). @johanneskoester if you want to tell me right before you merge (and we are sure others don't need to review) I can quickly update!

Reply:
Ok this is fixed! If anyone still wants to test, they can tweak to use the branch here.